src/js/content_scripts/input-mapper.js

```diff
@@ -186,6 +191,7 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
             currentTabIndex: 0
         },
         cutoffValue: 0.20,
+        multiplicationFactor: 50,
```

Comment:
"multiplication" is a bit generic, it'd be nice to find a better word or phrase to describe and distinguish this from `speedFactor`.